internal/config/config.go

```diff
@@ -45,10 +44,14 @@ type Config struct {
 	Database       DatabaseConfig       `json:"db,omitempty" mapstructure:"db"`
 	Meta           MetaConfig           `json:"meta,omitempty" mapstructure:"meta"`
 	Authentication AuthenticationConfig `json:"authentication,omitempty" mapstructure:"authentication"`
-	Warnings       []string             `json:"warnings,omitempty"`
```

Comment:
This does however have the side effect that `Warnings` will no longer be returned in the JSON output when hitting `/meta/config` endpoint..

Two thoughts here:
1. We could just remove it since it isn't documented anywhere
2. Or we could figure out another way to add these `Warnings` back in before marshalling the config as JSON

Reply:
I would lean on dropping them from that endpoint and letting warnings be logs only.
We probably could get away with it. Those warnings are meant to be ephemeral over time as you fix your config.
I wouldn't expect anything to be depending on their context for something behavioural.